astroquery/mpc/core.py

```diff
@@ -0,0 +1,289 @@
+from ..query import BaseQuery
+from . import conf
+from ..utils import async_to_sync
+
+__all__ = ['MPCClass']
+
+class InvalidArgumentException(Exception):
+    pass
+
+@async_to_sync
+class MPCClass(BaseQuery):
+    MPC_URL = 'http://' + conf.server + '/web_service/search_orbits'
+    # The authentication credentials for the MPC web service are publicly available and 
+    # can be openly viewed on the documentation page at https://minorplanetcenter.net/web_service/
+    MPC_USERNAME = 'mpc_ws'
+    MPC_PASSWORD = 'mpc!!ws'
+
+    def __init__(self):
+        super(MPCClass, self).__init__()
+
+    def query_object_async(self, *args, **kwargs):
+        """
+        Query around a specific object within a given mission catalog
+
+        Parameters
+        ----------
+        **kwargs :
```

Comment:
I'm not sure this extra level of indentation is necessary.  I also don't know if it will format correctly.  I think the docs below would be fine if just listed as normal function parameters, with the "The following..." paragraph put above "Parameters"